guides/basic-use/cli-commands.md

```diff
@@ -102,23 +102,51 @@ ember generate <type-of-file> <name-of-your-choice>
 
 ### What it does
 
-`ember generate` creates new files within your app. For example, you can use it to create components, routes, services, models, and more. For a full list, type `ember generate --help`.
+`ember generate` creates new files within your app. For example, you can use it to create components, routes, services, models, and more.
+
+For a full list, type **`ember generate --help`.**
 
 The CLI's `generate` command will ensure that new files contain the necessary boilerplate, that they go in the right directories, and that file naming conventions are followed. For example, components must always have a dash in their names.
 To avoid mistakes that are hard to debug, always use the CLI to create files instead of creating the files by hand.
 
-### Example use
+### Example Use: Component Generators
```

Comment:
I think this is not quite right, we should keep the "Example Use" header to be consistent with the rest of the page, but also because this is "An example of using *some* generator"